Include/httpClient/httpClient.h

```diff
@@ -121,7 +121,9 @@ typedef struct HCInitArgs {
     /// <summary>The Java Application Context.</summary>
     jobject applicationContext;
 } HCInitArgs;
-#elif HC_PLATFORM != HC_PLATFORM_NINTENDO_SWITCH
+#elif HC_PLATFORM == HC_PLATFORM_NINTENDO_SWITCH
+#include <httpClient/httpClient_switch.h>
+#elif
```

Comment:
I'd prefer to just have the client include this header.  The goal was that the NDA'd platforms would be completely additive, not requiring changes to the primary repo as much as possible.

Reply:
changed design